buildscripts/config/choose_modules.sh

```diff
@@ -4,7 +4,8 @@
 # http://www.apache.org/licenses/LICENSE-2.0.
 
 # Minimal JEDI Stack
-export      STACK_BUILD_CMAKE=Y
```

Comment:
Do we really want to build cmake ourselves? Every HPC system I know has a modern cmake module to load. macOS and Linux have new-ish cmake versions from brew or the native package managers.